packages/replayio/src/commands/record.ts

```diff
@@ -52,7 +55,12 @@ async function record(url: string = "about:blank") {
     logPromise(promise, {
       messages: {
         pending: "Uploading crash data...",
-        success: "Crash data uploaded successfully",
+        success: () => {
+          if (uploadableCrashes.some(recording => recording.uploadStatus === "failed")) {
+            return "Some of the crash data couldn't be uploaded";
```

Comment:
"✔ Crash data partially uploaded" - this doesn't like a failure to me so it could go a little bit unnoticed. Maybe something like "Crash data could only be partially uploaded"?

Either way, I'm fine with either - you have a better sense of English than I do :D

Reply:
Sure. I don't feel strongly about either one.